code/game/machinery/doors/multi_tile.dm

```diff
@@ -17,4 +19,23 @@
 	icon = 'icons/obj/doors/Door2x1glass.dmi'
 	opacity = 0
 	glass = 1
-	assembly_type = /obj/structure/door_assembly/multi_tile
\ No newline at end of file
+	assembly_type = /obj/structure/door_assembly/multi_tile
+
+/obj/machinery/door/airlock/multi_tile/setup_hatch()
+
+	if (overlays != null)
```

Comment:
safety. IIRC i ran into a problem somewhere, put that in as a temporary solution til i fixed it, saw no need to remove it. I'd rather have a door fail to initialise a hatch than throw a runtime error, though either should never happen